src/search.ts

```diff
@@ -55,11 +56,17 @@ export async function searchIndex(config: Config, effects = defaultEffects): Pro
     // implicitly a leading slash here.
     const id = file.slice(0, basename(file) === "index.md" ? -"index.md".length : -3);
 
-    const text = html
-      .replaceAll(/[\n\r]/g, " ")
-      .replaceAll(/<style\b.*<\/style\b[^>]*>/gi, " ")
-      .replaceAll(/<[^>]+>/g, " ")
-      .replaceAll(/\W+/g, " ");
+    // eslint-disable-next-line import/no-named-as-default-member
```

Comment:
Didn't understand why I could not `import {decode} from "he"` and had to do this instead.

Reply:
Yes, I haven’t been able to figure that out yet either. 😞